misc/landing-page/src/components/Info.astro

```diff
@@ -3,19 +3,19 @@ import Section from "./Section.astro";
 ---
 
 <Section
-    id="info"
-    background="dark"
-    class="flex flex-col gap-4 max-w-3xl font-bold text-3xl"
+  id="info"
+  background="dark"
+  class="flex flex-col gap-4 max-w-3xl font-bold md:text-[40px] text-2xl"
```

Comment:
We should use `em` for text, pick something close to 40px, same for other `[nnn-px]` values for text.